### ISSUE TITLE
Missing documentation in `cea.technologies.chiller_vapor_compression.calc_VCC`

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#3123]
I'm trying to figure out how `calc_VCC` works, and the function appears to be well documented... but it isn't:
```
def calc_VCC(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_chw_re_K, T_cw_in_K, VCC_chiller):
    """
    For th e operation of a Vapor-compressor chiller between a district cooling network and a condenser with fresh water
    to a cooling tower following [D.J. Swider, 2003]_.
    The physically based fundamental thermodynamic model(LR4) is implemented in this function.
    :type mdot_kgpers : float
    :param mdot_kgpers: plant supply mass flow rate to the district cooling network
    :type T_chw_sup_K : float
    :param T_chw_sup_K: plant supply temperature to DCN
    :type T_chw_re_K : float
    :param T_chw_re_K: plant return temperature from DCN
    :rtype Q_VCC_unit_size_W : float
    :returns Q_VCC_unit_size_W: chiller installed capacity
    ..[D.J. Swider, 2003] D.J. Swider (2003). A comparison of empirically based steady-state models for
    vapor-compression liquid chillers. Applied Thermal Engineering.
    """
```

Some of these undocumented variables are pretty clear (`peak_cooling_load`, `q_chw_load_Wh`) but what is `Tcw_in_K`? Gitblame tells me @shanshanhsieh originally wrote the function itself and documented, then @emanuel030 added some variables and functions but didn't update the documentation.

This is a problem, because researchers come but their poorly documented functions stay!

@MatNif since you're working on modifying some of the code in CEA, I strongly advise you to document every function you add :) And to everyone else: I strongly advise requiring good documentation before merging PRs.

Oh, by the way, `T_cw_in_K` appears to be the source temperature. 
```
VCC_operation = chiller_vapor_compression.calc_VCC(chiller_size, Qc_from_VCC_W, T_DCN_sup_K, T_DCN_re_K, **T_source_K**, VCC_chiller)
```
But you probably shouldn't need to go into the optimization code base in order to understand a function in the `technologies` code base.

**Comment by MatNif** [comment]
Very good point @martin-mosteiro. I'm currently working on a refactoring/documentation patch for a lot of the functions that are being called within the optimization-routine: https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/Refactoring-of-Optimization-Algorithm. 
If anyone wants to add to it feel free to do so.

**Comment by shanshanhsieh** [comment]
this has been addressed in PR #3132
